rust/sdk/examples/query_bounds.rs

```diff
@@ -23,6 +23,7 @@ async fn main() -> anyhow::Result<()> {
             &serde_json::json!({ "temperature": 40 }),
         )?
         .await?;
+    println!("{:?}", publish_response);
```

Comment:
```suggestion
```

Reply:
a few lines down is a very similar `println!()`; this uses the `publish_response` which would otherwise need to be renamed to avoid the unused variable warning

Reply:
@jmg-duarte